kapo_ordnungsbussen/src/etl.py

```diff
@@ -34,7 +39,7 @@ def main():
     df_all['Ü-Ort PLZ'] = df_all['Ü-Ort PLZ'].replace(credentials.plz_replacements).astype(int)
 
     logging.info(f'Replacing old BuZi with new ones using lookup table...')
-    df_lookup = pd.read_excel(os.path.join(credentials.data_orig_path, '2021_06_30', 'Lookup-Tabelle BuZi.xlsx'))
+    df_lookup = pd.read_excel(os.path.join(credentials.data_orig_path, '2022_06_30', 'Lookup-Tabelle BuZi.xlsx'))
```

Comment:
@HesterFrederiek I think it should be left at '2021_06_30' - let's quickly discuss, I'll call you...